workflows/config_init_test.go

```diff
@@ -19,6 +19,33 @@ func TestNewConfigInitializer(t *testing.T) {
 	assert.NotNil(lister)
 }
 
+func TestSubstituteEnvironmentVariables(t *testing.T) {
```

Comment:
This test should be moved to the `common` in `context_test.go` since it is testing code from `context.go`

Reply:
moved...